lib/screens/v2/widget_instance/alert.ex

```diff
@@ -96,18 +96,26 @@ defmodule Screens.V2.WidgetInstance.Alert do
   end
 
   defp serialize_route_pills(t) do
-    t
-    |> informed_routes()
-    |> Enum.to_list()
-    |> Enum.sort_by(fn route_id ->
-      case Integer.parse(route_id) do
-        # Bus route (including SL_, CT_)
-        {route_number, ""} -> route_number
-        # Non-bus route
-        _ -> route_id
-      end
-    end)
-    |> Enum.map(&RoutePill.serialize_for_alert/1)
+    routes = informed_routes(t)
+
+    if MapSet.size(routes) <= 3 do
+      routes
+      |> Enum.to_list()
+      |> Enum.sort_by(fn route_id ->
```

Comment:
You can consider this out of scope for this PR, but routes do have a sort order which represents the order in which they should be shown to riders. (I don't think we're parsing this from the routes API response, currently.)

Reply:
Oh, good to know. I'll create a followup task.